NAMESPACE

```diff
@@ -206,3 +206,5 @@ S3method(format_list_item, data.frame)
 
 export(fdroplevels, setdroplevels)
 S3method(droplevels, data.table)
+
+S3method(sort_by, data.table)
```

Comment:
This needs to be conditioned on the R version, right?

Reply:
Absolutely. I'll add a version guard now.